vignettes/vignette.rmd

```diff
@@ -20,10 +20,7 @@ Maps are converted to data.frames with additional class "map".  Map objects in R
 For more information on msgpack types, see [here](https://github.com/msgpack/msgpack/blob/master/spec.md).  
 
 ### Installation:
-1. On Windows, install Rtools (https://cran.r-project.org/bin/windows/Rtools/)
-2. In R, install devtools: `install.packages("devtools")`
-3. `library(devtools)`
-4. `install_github("traversc/msgpack2R")`
+1. `Install.packages("RcppMsgPack")`

```

Comment:
Same here.

Was about to make the same changes re Windows.  Should be easier once Uwe's builder kicks in.